### ISSUE TITLE
Email notification that feedback on a plan is complete is not sent

### ISSUE THREAD

**Issue by mariapraetzellis** [DMPRoadmap/roadmap#2521]
**Expected behaviour:**

Clicking on "Complete" on the Plans Notifications table should send an email to the plan creator notifying the plan owner that the feedback is now available.

![screenshot-dmptool org-2020 05 21-15_27_34](https://user-images.githubusercontent.com/27511575/82612427-ad5a2100-9b77-11ea-91b2-ea532e00ab26.png)

**Actual behaviour:**
No email is sent.

**Steps to reproduce:**

1. I requested feedback on this plan: http://dmptool.org/plans/51258/
2. Went to Plans page and marked as Complete
3. Checked email to see if I got notification (but didn't)

**I tested this on DMPTool production, however @sjDCC mentioned this was broken on DMPOnline test, so I think it's a Roadmap issue.

**Comment by briri** [comment]
The PR to fix this should be made to `master`. We can backfill into `development` afterward

**Comment by briri** [comment]
related to #2527

**Comment by magdalenadrafiova** [comment]
Sam confirmed this works for DMPonline - so it might be a problem on your end - @raycarrick-ed to still confirm today just to double check

**Comment by raycarrick-ed** [comment]
Have double checked: created plan, requested feedback, marked feedback as complete and got email. This was tested on DMPonline.

**Comment by sjDCC** [comment]
I just checked too in case it was just an Edinburgh email exception. 

Received original notification that plan had been submitted for review.
![feedback-requested](https://user-images.githubusercontent.com/6169940/83152215-76df5180-a0f5-11ea-9204-4bf4913eb4a1.PNG)

Saw it in list as expected
![test feedback](https://user-images.githubusercontent.com/6169940/83152229-7ba40580-a0f5-11ea-93e1-54e69955b474.PNG)

And then as user (s.jones@arts) I received notice when 'complete' had been ticked
![feedback-complete](https://user-images.githubusercontent.com/6169940/83152242-7fd02300-a0f5-11ea-9a00-8e5c28523997.PNG)

Possibly something on mailer config?

**Comment by briri** [comment]
yes most likely something on our end then. I will have a look today

**Comment by briri** [comment]
Ok, so I tracked this down to the email being blocked by our AWS SES rules. I patched for our DMPTool fork but think this should be changed in DMPRoadmap as well as any other installations with similar email server rules would encounter the same issue.

This one particular email tries to use the Org's contact email as the 'from' sender address:
```ruby
    from: requestor.org.contact_email,
```
This is a really bad practice. We are in essence sending out email on behalf of the Org contact :/
The fix would be to remove that line and allow it to use the default which is the email defined in our `branding.yml` file (e.g. 'do-not-reply@my-server.org')
